ssl/t1_lib.c

```diff
@@ -1318,7 +1318,7 @@ TICKET_RETURN tls_decrypt_ticket(SSL *s, const unsigned char *etick,
         /* Some additional consistency checks */
         if (p != sdec + slen || sess->session_id_length != 0) {
```

Comment:
Is this well defined behaviour? We seem to be doing pointer arithmetic on a pointer that was freed a couple of lines above.